sktime/forecasting/structural.py

```diff
@@ -319,6 +325,59 @@ def _fit_forecaster(self, y, X=None):
             low_memory=self.low_memory,
         )
 
+    def _predict_interval(self, fh, X=None, coverage=0.90):
```

Comment:
according to the extension template, this should be
`    def _predict_quantiles(self, fh, X=None, alpha=None):`

Reply:
thanks for the input! I changed the implementation in https://github.com/alan-turing-institute/sktime/pull/2454/commits/5d069bb1d0b39c3c593c9dba06321f4d21e7f419. Now I am writing some unit-test because I think the current implementation handles `exog` variables correctly (that is why I this one still a draft 😅 ).

I am also refactoring the tests for this estimators because I have gained a bit more experience writing nicer  pytests :)

Reply:
great!

Although, now they fail...